src/MOLECULE/angle_cosine.cpp

```diff
@@ -80,6 +84,7 @@ void AngleCosine::compute(int eflag, int vflag)
     delx2 = x[i3][0] - x[i2][0];
     dely2 = x[i3][1] - x[i2][1];
     delz2 = x[i3][2] - x[i2][2];
+    domain->minimum_image(delx2, dely2, delz2);
```

Comment:
These should be removed, this is not how PBC's are handled in LAMMPS. Instead, it is done in the NTopo classes e.g. atom1 = domain->closest_image(i,atom1);